From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Alexandre Derumier <aderumier@odiso.com>
Date: Mon, 14 May 2018 16:27:22 +0200
Subject: [PATCH] addons scripts : fix ENV for interfaces options

This is a bug currently, where env vars with interfaces options
were not reset between interfaces

https://github.com/CumulusNetworks/ifupdown2/issues/49
---
 ifupdown2/ifupdown/ifupdownmain.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ifupdown2/ifupdown/ifupdownmain.py b/ifupdown2/ifupdown/ifupdownmain.py
index 2daf49f..4931f73 100644
--- a/ifupdown2/ifupdown/ifupdownmain.py
+++ b/ifupdown2/ifupdown/ifupdownmain.py
@@ -1552,7 +1552,7 @@ class ifupdownMain(ifupdownBase):
         cenv = None
         iface_env = ifaceobj.get_env()
         if iface_env:
-            cenv = os.environ
+            cenv = dict(os.environ)
             if cenv:
                 cenv.update(iface_env)
             else:
-- 
2.11.0

